src/Tizen.Multimedia.MediaPlayer/Player/AudioEffect.cs

```diff
@@ -135,9 +150,13 @@ public Range BandLevelRange
         /// <summary>
         /// Gets the value whether the AudioEffect is available or not.
         /// </summary>
+        /// <remarks>This function returns the availability of the audio effect function group and
+        /// it could be unavailable depending on the platform capabilities.</remarks>
         /// <exception cref="InvalidOperationException">
         ///     If audio offload is enabled by calling <see cref="AudioOffload.IsEnabled"/>. (Since tizen 6.0)
         /// </exception>
+        /// <exception cref="NotAvailableException">The function is not available depending on the audio codec type. (Since tizen 6.0)</exception>
```

Comment:
This sentence is not bad.
However, how about the following sentence?

 /// <exception cref="NotAvailableException">Depending on the audio codec type, the function is not available. (Since tizen 6.0)

If you think the suggested phrase is better, please use it for others.